ajax/libs/labelauty/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "jquery-labelauty",
+  "version": "1.1.0",
+  "author": "Francisco Neves <contact@francisconeves.com>",
+  "repository": {
+    "type":"git",
+    "url":"git://github.com/fntneves/jquery-labelauty"
```

Comment:
There should be a space after colon mark. https://github.com/cdnjs/cdnjs/blob/5ae759bcf8e719734c6453a5b4feebc7d9832e3d/ajax/libs/labelauty/package.json#L6-#L7